server/app/schema.py

```diff
@@ -1560,6 +1585,14 @@ def resolve_classic(self, info):
         ),
     )
 
+    user_by_name = graphene.Field(User, username=graphene.String(required=True))
+
+    def resolve_user_by_name(self, info, username):
+        return ModelUserAccount.find_by_username(username)
+
+    def resolve_set_count(self):
```

Comment:
Hmm, what's this?

Reply:
The query to pull the number of sets a user has